spec/controllers/nsm/steps/disbursements_summary_controller_spec.rb

```diff
@@ -20,6 +20,17 @@
       expect(controller.instance_variable_get(:@items_incomplete_flash)[:default])
         .to include('2 disbursements are incomplete:')
     end
+
+    it 'creates the correct link for incomplete disbursement' do
+      incomplete_disbursements = claim.disbursements.reject(&:complete?).sort_by(&:position)
+      get :edit, params: { id: claim.id }
+      expect(controller.instance_variable_get(:@items_incomplete_flash)[:default])
+        .to include("<a class=\"govuk-link\" href=\"/non-standard-magistrates/applications/#{claim.id}" \
```

Comment:
You could make it a bit more readable by just checking for `application/#{claim.id}`

Reply:
I wanted to ensure the index of the items consistently paired with the right item id in this test